examples/main/take_picture.c

```diff
@@ -142,6 +142,10 @@ static esp_err_t init_camera()
 
 void app_main()
 {
+#if CONFIG_IDF_TARGET_ESP32C2
+    ESP_LOGE(TAG, "Camera not supported on esp32c2 chip");
```

Comment:
The way this line is written, it will have to be updated for every chip which doesn't have camera interface.

Can we write it in a way that will reduce the follow-up work, instead?

For example:

1. Introduce a hidden Kconfig option which will be true for all targets where the camera is supported (if we don't have such an option yet)
2. Check for the inverse of this option here